pipelines/xDAWN+LDA.yml

```
@@ -0,0 +1,14 @@
+name: xDAWN + LDA
+
+paradigms:
+  - P300
+
+citations:
+  - https://doi.org/10.1109/TBME.2009.2012869
+
+pipeline:
+  - name: Xdawn
+    from: pyriemann.spatialfilters
+
+  - name: LinearDiscriminantAnalysis
+    from: sklearn.discriminant_analysis
```

```suggestion
pipeline:
  - name: Xdawn
    from: pyriemann.spatialfilters

  - name: Vectorizer
    from: mne.decoding

  - name: LinearDiscriminantAnalysis
    from: sklearn.discriminant_analysis
```
You need to add a vectorizer to pass the XDAWN components to the LDA, as in this [example](https://github.com/NeuroTechX/moabb/blob/6047c8dff0259c7664218f62e414be59e2945ae7/examples/plot_within_session_p300.py#L52). 

I think it could also be a good idea to use this PR to update the examples that redefine a Vectorizer class instead of reusing the one proposed in MNE. I found that those examples could be modified :
- [plot_within_session_p300.py](https://github.com/NeuroTechX/moabb/blob/develop/examples/plot_within_session_p300.py#L52)
- [plot_learning_curve_p300.py](https://github.com/NeuroTechX/moabb/blob/develop/examples/learning_curve/plot_learning_curve_p300.py#L54)